Contributing.md

```diff
@@ -0,0 +1,18 @@
+# CONTRIBUTING
```

Comment:
We should probably open a separate PR to update this file 😄 Currently it's pretty barebones.

Reply:
Just realized that because I merged the latest dev into the msalruntime-broker branch it's showing a lot of unecessary file changes... This file and most of the other changes have already been committed into dev, the branch that this PR is being merged into was just quite behind.

For the API changes that you might be more interested in, see this commit: https://github.com/AzureAD/microsoft-authentication-library-for-java/pull/733/commits/b315b3e970a6b2dce3d0c34fe0070c8389f67b5f